src/components/scrollBar/scrollBar.api.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "ScrollBar",
+  "category": "lists",
+  "description": "Scrollable container with animated gradient overlay for horizontal scroll",
+  "example": "https://github.com/wix/react-native-ui-lib/blob/master/demo/src/screens/componentScreens/ScrollBarScreen.js",
+  "props": [
+    {
+      "name": "useList",
+      "type": "boolean",
+      "description": "Whether to use a FlatList. NOTE: you must pass 'data' and 'renderItem' props as well"
+    },
+    {
+      "name": "containerView",
+      "type": "React.ComponentClass",
+      "description": "The element to use as a container, instead of a View"
+    },
+    {
+      "name": "containerProps",
+      "type": "object",
+      "description": "The props to pass the container"
+    },
+    {
+      "name": "height",
+      "type": "number",
+      "description": "The component's height"
+    },
+    {
+      "name": "gradientHeight",
+      "type": "number",
+      "description": "The gradient's height, defaults to the component's height"
+    },
+    {
+      "name": "gradientWidth",
+      "type": "number",
+      "description": "The gradient's width"
+    },
+    {
+      "name": "gradientMargins",
+      "type": "number",
+      "description": "The gradient's margins for the edge"
+    },
+    {
+      "name": "gradientColor",
+      "type": "string",
+      "description": "The gradient's tint color"
+    },
+    {
+      "name": "gradientImage",
+      "type": "ImageSourcePropType",
+      "description": "The gradient's image, instead of the default image"
+    },
+    {
+      "name": "focusIndex",
+      "type": "number",
+      "description": "The index to currently focus on"
+    }
+  ],
+  "snippet": [
+    "<ScrollBar>",
```

Comment:
ScrollBar needs more items to be scrollable. Please consider using map to create more items

Reply:
You removed the file instead of fixing the snippet. Please restore the file with the fix

Reply:
ScrollBar displayName is not IGNORE, see line 78 (you're confusing Item.displayName in line 278)

Reply:
Please fix snippet to have more items